src/doc/rustdoc/src/write-documentation/documentation-tests.md

```diff
@@ -317,7 +317,7 @@ only shows the part you care about.
 
 ```rust
 /// ```ignore
-/// fn foo() {
+/// fn foo() {}
```

Comment:
this is probably on purpose, being invalid rust code

Reply:
Thanks~